.github/workflows/docs_deploy.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
I will probably make some minor changes to make the workflow more uniform to my existing ones, but looks good. :)